code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1243,7 +1243,7 @@
 	name = "Syntiflesh"
 	id = "syntiflesh"
 	result = null
-	required_reagents = list("blood" = 5, "clonexadone" = 5)
+	required_reagents = list("blood" = 5, "clonexadone" = 1) //VOREStation Edit

```

Comment:
This entire datum field can be put into a Chemistry-Recipes_vr.dm file.

Or, at least, what you are changing.

Remove the vorestation edit line and do this instead in the new file:
```
/datum/chemical_reaction/food/syntiflesh
     required_reagents = list("blood" = 5, "clonexadone" = 1)```